test-cloud/pom.xml

```diff
@@ -156,6 +160,15 @@
     </profile>
     <profile>
       <id>jbpm</id>
+      <properties>
+        <kie.app.template.workbench>${templates.jbpm.dir.url}/rhpam70-businesscentral.yaml</kie.app.template.workbench>
+        <kie.app.template.workbench-monitoring>${templates.jbpm.dir.url}/rhpam70-businesscentral-monitoring.yaml</kie.app.template.workbench-monitoring>
+        <kie.app.template.smartrouter>${templates.jbpm.dir.url}/rhpam70-smartrouter.yaml</kie.app.template.smartrouter>
+        <kie.app.template.controller>${templates.jbpm.dir.url}/rhpam70-controller.yaml</kie.app.template.controller>
+        <kie.app.template.kie-server.database>${templates.jbpm.dir.url}/rhpam70-kieserver-postgresql.yaml</kie.app.template.kie-server.database>
```

Comment:
This template is available as official one, I think we should take it from there.